http/swagger.yml

```diff
@@ -9239,24 +9237,49 @@ components:
       allOf:
         - $ref: "#/components/schemas/NotificationEndpointBase"
         - type: object
-    SMTPNotificationEndpoint:
-      type: object
-      allOf:
-        - $ref: "#/components/schemas/NotificationEndpointBase"
-        - type: object
+          required: [url, token]
+          properties:
+            url:
+              type: string
+            token:
+              type: string
     PagerDutyNotificationEndpoint:
       type: object
       allOf:
         - $ref: "#/components/schemas/NotificationEndpointBase"
         - type: object
+          required: [url, routing-key]
+          properties:
+            url:
+              type: string
+            token:
+              type: string
     WebhookNotificationEndpoint:
       type: object
       allOf:
         - $ref: "#/components/schemas/NotificationEndpointBase"
         - type: object
+          required: [url]
```

Comment:
`method` and `authmethod` should be required

Reply:
swagger changes is passed to @121watts , so he won't be blocked.